packages/rocketchat-ui/client/components/tabs.js

```diff
@@ -0,0 +1,18 @@
+Template.tabs.onRendered(function() {
+	this.$('.tab').first().addClass('active');
+	this.data.tabs.onChange(this.data.tabs.tabs[0].value);
+});
+
+Template.tabs.events({
+	'click .tab'(e) {
+		$('.tab').removeClass('active');
```

Comment:
I think this way will not work for nested tabs

Reply:
how nested tabs should work? We will have nested tabs? We already have a design for that?

Reply:
we dont have design to show you now, but just to think...not for now, but is a small change

Reply:
Can we do a simple `{activeClass}` variable on the helper and change according to state?

Reply:
I created a 'ReactiveVar' to keep the state of the active tab and adde a helper to check which tab is active and add an active class to this tab.

Thank you for your suggestions :)